example/bcachefs-subvol.nix

```diff
@@ -0,0 +1,49 @@
+{
+  disko.devices = {
+    disk = {
+      vdb = {
+        device = "/dev/disk/by-path/pci-0000:02:00.0-nvme-1";
+        type = "disk";
+        content = {
+          type = "gpt";
+          partitions = {
+            ESP = {
+              end = "500M";
+              type = "EF00";
+              content = {
+                type = "filesystem";
+                format = "vfat";
+                mountpoint = "/boot";
+              };
+            };
+            root = {
+              name = "root";
+              end = "-0";
+              content = {
+                type = "bcachefs";
```

Comment:
If we have two types, than we should rename this one to `bcachefs_single` or similar. Otherwise it creates confusion when people try to use `bcachefs` for multi disk setups.